Sources/KeyboardShortcuts/Utilities.swift

```diff
@@ -559,3 +559,104 @@ extension Character {
 		self = Character(content)
 	}
 }
+
+final class UserDefaultsObservation: NSObject {
```

Comment:
This class does not need to know about `KeyboardShortcuts.Name`. It can be more general.

Reply:
Agree